images/macos/provision/core/git.sh

```diff
@@ -8,6 +8,11 @@ brew extract --version=2.35.1 git local/git
 brew install git@2.35.1
```

Comment:
Should we remove this workaround?

Reply:
for sure, this was one of the goal @AlexanderGitin

Reply:
I removed temporary fix please review